packages/aws-cdk-lib/aws-events-targets/README.md

```diff
@@ -337,3 +337,33 @@ rule.addTarget(new targets.EventBus(
   ),
 ));
 ```
+
+## Run an ECS Task
```

Comment:
Can you add a little section here about tags? Based on the conversation so far
on this PR I think it would help users to understand the different types of tags
and how they are used.

Reply:
Yep that sounds good, I'm not sure why GitHub isn't showing this diff as outdated now but I've gone ahead and added some clarifying notes about tagging here.